### PR TITLE
Added Postal code DK

### PR DESCRIPTION
**Pull request by BenjaminHoegh**
This accept all danish(DK) zip codes (1000 to 9999).
 also work for the danish kingdoms Greenland(use same digital zip codes).
 Not work for the danish kingdom Faroe Islands after they use 3 digi zip codes.

Note: Greenland and Faroe Islands is a part of the Danish Kingdom and not the country it self.

### CODE REVIEW

File: src/additional/postalcodeDK.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* Matches Danish postcode */
+$.validator.addMethod( "postalcodeDK", function( value, element ) {
+	return this.optional( element ) || /^[0-9]{4}$/.test( value );
+}, "Please specify a valid postal code" );
+
+/*
+ * This accept all danish(DK) zip codes (1000 to 9999)
```
Comment (staabm) [inline src/additional/postalcodeDK.js]: Please move this comment to the top

File: src/additional/postalcodeDK.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/* Matches Danish postcode */
```
Comment (staabm) [inline src/additional/postalcodeDK.js]: Replace with the extensive comment from below

**Comment by staabm** [comment]
Please add tests

**Comment by RobJohnston** [comment]
Optionally, you _could_ chage the regex to `^[0-9]{3,4}$` to get it to work with the Faroe Islands as well.

**Comment by Arkni** [comment]
Hi @HawkonDK 
Are you willing to continue working on this?

Thanks!

File: src/additional/postalcodeDK.js (hunk 1 above)
Comment (BenjaminHoegh) [reply src/additional/postalcodeDK.js]: Oh! why did I not see that? a bigger question is why GitHub didn't notify me about your feedback

**Comment by BenjaminHoegh** [comment]
Yes @Arkni Github for some reason didn't notify me about your comments

**Pull request closed without merging**